general/management/commands/gm_worker_async_tasks.py

```diff
@@ -90,3 +92,17 @@ def task_whitelist_user(self, gearman_worker, gearman_job):
                 count_done = count_done + 1
             self.write_stdout("Finished processing one ticket, %d remaining" % (len(tickets)-count_done))
         return 'true' if len(tickets) == count_done else 'false'
+
+    def task_email_random_sound(self, gearman_worker, gearman_job):
+        self.write_stdout("Notifying user of random sound of the day")
```

Comment:
I would only care for the 3rd point, but I'd argue this is not a problem of this issue but more of the general concept of random sound of the day. modularsamples will get one email per week, as he uploaded 1/7 of the whole freesound contents. I think we should maybe add some constrains to the way we choose the random sound, but for now I think this PR is ok.